Convex_hull_3/doc/Convex_hull_3/Concepts/ConvexHullTraits_3.h

```diff
@@ -84,6 +84,13 @@ Predicate object type that provides
 `q` is on the positive side of the halfspace `h`. 
 */ 
 typedef unspecified_type Has_on_positive_side_3; 
+  
+  /*!
+  Predicate object type that provides 
+  `bool operator()(Plane_3 h, Point_3 q)`, which determines if the point 
+  `q` is on the negative side of the halfspace `h`. 
+  */ 
+  typedef unspecified_type Has_on_negative_side_3; 
```

Comment:
Now the traits requires both `Has_on_positive_side_3` and `Has_on_negative_side_3`. I suggest that both are replaced by [`Oriented_side_3`](https://doc.cgal.org/latest/Kernel_23/classKernel_1_1OrientedSide__3.html) that provides:
> `Oriented_side operator() (const Kernel::Plane_3 &h, const Kernel::Point_3 &p)`

Reply:
@sloriot, do you agree ?

Reply:
No, I'm working on it